packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div>
+        <p
+            :class="{
+                [$style['c-restaurantCard-iconText']]: true,
+                [$style['c-restaurantCard-iconText--bold']]: isBold
+            }"
+            aria-hidden="true"
+            :data-test-id="dataTestId">
+            <span
+                v-if="showIcon"
+                :class="{
+                    [$style['c-restaurantCard-iconText-icon']]: true,
+                    [$style['c-restaurantCard-iconText-icon--hideOnMidBellow']]: hideIconInTileView
+                }"
+                :style="`fill:${color};`"
+                data-test-id="icon-text-icon">
+                <slot />
+            </span>
+            <span
+                :class="$style['c-restaurantCard-iconText-content']"
+                :style="`color:${color};`"
+                data-test-id="icon-text-visible-text">
+                {{ text }}
+            </span>
+        </p>
+        <span
+            class="is-visuallyHidden"
+            data-test-id="icon-text-accessible-text">
+            {{ accessibleText || text }}
+        </span>
```

Comment:
I don't think we currently do this on SearchWeb, and we're probably not going to start generating new copy as part of this work - we'd need to get new string values for all tenants which is out of scope. So we can probably leave this alone for now as it isn't something we've had as a ticket from accessibility testing. 

If it's something we want to add in future, then we probably want to do that as a separate piece of work - we'll need to work with copywriters etc.

Reply:
<img width="457" alt="Screenshot 2021-11-04 at 09 07 07" src="https://user-images.githubusercontent.com/16766185/140286935-e33943d8-a7d0-4fca-b2a3-8bc99b318729.png">

for context - the current implementation

Reply:
I'd also be keen to know if this could be achieved via aria attributes as opposed to adding more DOM nodes to the page

Reply:
We do this on HomeWeb, and I don't want to make the home page less accessible as a result of this work. For SearchWeb we can use the regular text for now, which is reflected in the code: If there's no accessible text - we use the regular text for screen readers. It's a huge difference for screen reader users to hear "1 mile" vs "1 mile from your location".

Reply:
So then I'd suggest 2 things - 

- only render this span if there is accessible text; there's no point in duplicating the same text 
- create an internal ticket so that we don't forget that we now need these translations available in searchweb

Reply:
> only render this span if there is accessible text; there's no point in duplicating the same text

if there's no accessible text, we should at least use regular text. Otherwise, this information will be missed completely from the screen readers. Or we need some logic inside `aria-hidden="true"` which will only make the component less readable. Happy to hop on a call and discuss.

Reply:
We could potentially use Vue to add/remove the aria-hidden in that case 🤔

Reply:
ticket to address the accessibility of the component https://justeattakeaway.atlassian.net/browse/SEARCH-1996